front/src/routes/integration/all/tuya/actions.js

```diff
@@ -0,0 +1,209 @@
+import { RequestStatus } from '../../../../utils/consts';
+import update from 'immutability-helper';
+import debounce from 'debounce';
+import createActionsIntegration from '../../../../actions/integration';
+
+function createActions(store) {
```

Comment:
Could we put those functions in the class instead of using unistore? We try to avoid this now :) ( https://community.gladysassistant.com/t/frontend-amelioration-diverses-experience-developpement/8125 )